packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -0,0 +1,54 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { StringField } from '../../table/types';
+import { TableColumn } from '../base';
+import { styles } from '../base/styles';
+import { template } from '../base/template';
+import { cellStyles } from './styles';
+import { cellTemplate } from './template';
+
+export type TableColumnTextCellData = StringField<'value'>;
+export interface TableColumnTextColumnConfig {
+    placeholder: string;
+}
+
+/**
+ * The table column for displaying strings.
+ */
+export class TableColumnText extends TableColumn<
+TableColumnTextCellData,
+TableColumnTextColumnConfig
+> {
+    public cellStateDataFieldNames = ['value'] as const;
+
+    @attr({ attribute: 'field-name' })
+    public fieldName!: string;
+
+    @attr
+    public placeholder?: string;
+
+    public override readonly cellStyles = cellStyles;
```

Comment:
Any reason this is the only member marked `override`?

Reply:
It's because the base class definition isn't abstract. This would be the other benefit to marking this member as abstract, even if the ultimate implementation should be empty. I don't feel strongly either way at the moment.

Reply:
Marked as abstract.